### ISSUE TITLE
Hub Watcher removes newly joined gateway..

### ISSUE THREAD

**Issue by jihoonl** [robotics-in-concert/rocon_multimaster#284]
```
[WARN] [WallTime: 1409881063.688007] Hub Watcher: gateway dangeun has been unavailable for 10 seconds! Marking as unavailable.
[WARN] [WallTime: 1409881063.688417] Hub Watcher: gateway dangeun has been unavailable for 7200.0 seconds! Removing from hub.
[WARN] [WallTime: 1409881064.141839] Gateway : requested gateway info for unavailable gateway [dangeun]
```

```
[WARN] [WallTime: 1409881065.143444] Gateway : requested gateway info for unavailable gateway [dangeun]
[WARN] [WallTime: 1409881066.145748] Gateway : requested gateway info for unavailable gateway [dangeun]
[WARN] [WallTime: 1409881067.147837] Gateway : requested gateway info for unavailable gateway [dangeun]
[WARN] [WallTime: 1409881068.150025] Gateway : requested gateway info for unavailable gateway [dangeun]
[WARN] [WallTime: 1409881069.152437] Gateway : requested gateway info for unavailable gateway [dangeun]
```

**Comment by stonier** [comment]
There is a tertiary state for clients from the view of the watcher :+1: 
- available
- unavailable (after 10s)
- obso (+7200s)

I think these values are already parameterised, but if not parameterise them.

From the above it looks like the logic for going from unavailable to obso is broken in the hub watcher.

**Comment by stonier** [comment]
Solved in #287, it was a redis return code causing the problem.
